src/Styleguide/Components/Tabs.tsx

```diff
@@ -185,3 +185,15 @@ const TabContainer = styled.div`
 const ActiveTabContainer = styled.div`
   ${styles.activeTabContainer};
 `
+
+const TabSuperScriptWrapper = styled.sup`
+  margin-left: 2px;
+`
+
+export const TabSuperscript = ({ children }: { children?: any }) => (
```

Comment:
It might be nicer if this was exported as `<Sup>` as a play on real html tag `<sup>`. Then if we ever need a sub tab we could add `<Sub>`.

Reply:
Also, `any` can be typed as `JSX.Element`

Reply:
Another option to type the children would be like:

```ts
export const TabSuperscript: React.SFC<{}> = ({ children }) => ( ... )
```